usr/lib/hsm_mk_change/hsm_mk_change.c

```diff
@@ -49,11 +48,11 @@ CK_RV hsm_mk_change_lock_create(void)
     mode_t mode = (S_IRUSR | S_IRGRP);
 
     if (hsm_mk_change_lock_fd == -1)
-        hsm_mk_change_lock_fd = open(OCK_HSM_MK_CHANGE_LOCK_FILE, O_RDONLY);
+        hsm_mk_change_lock_fd = open(OCK_HSM_MK_CHANGE_LOCK_FILE, O_RDWR);
```

Comment:
At other places you have WRONLY ? I don't care if WRONLY or RDWR, but I guess we should use all the same at every place we are opening a lock file.